src/main/java/com/splunk/logging/AckManager.java

```diff
@@ -39,26 +39,52 @@
 
   private static final ObjectMapper mapper = new ObjectMapper();
   private final HttpEventCollectorSender sender;
-  private final AckPollScheduler ackPollController = new AckPollScheduler();
+  private final PollScheduler ackPollController = new PollScheduler();
```

Comment:
Well, since in this branch we did some refactoring so that all types of polling are done using a single type of PollScheduler, it does become harder to tell what is going on (which was my knock on this refactor in the first place). We will have a pollScheduler variable in several places. We will not be able to tell from the variable, what that scheduler is polling. From my perspective this is not an improvements. There is alway some tension between DRY and "can I tell what is going on, or at least where to look in the code, by looking at classnames and variable names". IMHO we are decreasing the ability to understand the code from a quick glance at the class names, in order to save a few lines of code. Just my $0.02.